alcs-api/src/queues/scheduler/scheduler.service.ts

```diff
@@ -0,0 +1,16 @@
+import { InjectQueue } from '@nestjs/bull';
+import { Injectable } from '@nestjs/common';
+import { Queue } from 'bull';
+import { CRON_EXPRESSIONS } from '../../common/constant';
+
+@Injectable()
```

Comment:
You might be able to just put the CRON expression const here until we have enough it needs its own file?